sites/public/pages/applications/financial/income.tsx

```diff
@@ -207,6 +207,8 @@ const ApplicationIncome = () => {
                 validation={{ required: true }}
                 fields={incomePeriodValues}
                 dataTestId={"app-income-period"}
+                fieldGroupClassName="grid grid-cols-1"
```

Comment:
In the application, two radio buttons should be displayed vertically (regression, they are currently displayed horizontally)